src/components/Header/Navbar.tsx

```diff
@@ -137,3 +138,8 @@ const secondaryStyle = css`
 const SecondaryLink = styled(StyledLink)`
   ${secondaryStyle}
 `;
+
+const ChristmasTreeIcon = styled.img`
+  height: 32px;
+  width: 32px;
```

Comment:
It's a tad small; I think you could size up to `44px`. 

If you want it to be even bigger, you can add `position: relative` and move the icon up a little. Example:

<img width="160" alt="Screen Shot 2022-12-20 at 11 38 05 AM" src="https://user-images.githubusercontent.com/47064971/208718782-64e3651b-8d85-4268-ad9c-82723447d65c.png">